packages/binding/src/entityEvents/useDerivedField.ts

```diff
@@ -48,8 +48,7 @@ export const useDerivedField = <SourceValue extends FieldValue = FieldValue>(
 					return
 				}
 
-				const transformedValue = transform(sourceAccessor.value)
-				derivedAccessor.updateValue(transformedValue, {
+				derivedAccessor.updateValue(sourceAccessor.value === null ? null : transform(sourceAccessor.value), {
```

Comment:
This is quite a change in behaviour. 
a) If you want go this route, change the type on line 20 in this file.
b) But I would prefer to do it in `SlugField` - handle there the case when driver field is null and return null in that case.

Reply:
@matej21 can you respond, since you proposed this change?

Reply:
IMHO, when we have schema exposed on the field done in #168, we could also check `nullable: true` in the parser.

1. Should parser return `null` only when `null` is allowed?
1. Should parser return `null` when the value on server is still `null` even though `nullable: false`?
1. What should be returned when parsing was not possible?

Which of these should parser have resposibility to implement?

Reply:
Yep, doing this in slugfield is a also possible (I've proposed this alternative approach in that slack message). I'm ok with both solutions.